.gitignore

```diff
@@ -201,4 +203,7 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
-# End of https://www.toptal.com/developers/gitignore/api/windows,visualstudiocode,node,cypressio
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/windows,visualstudiocode,node,cypressio
+
+# Github
+.github/*
```

Comment:
why you are adding .github as ignored path? this is not required or needed, and will bring you problems in the future